common/tools/dev-tool/package.json

```diff
@@ -69,6 +69,7 @@
     "rollup": "^1.16.3",
     "rollup-plugin-sourcemaps": "^0.4.2",
     "rollup-plugin-visualizer": "^4.0.4",
-    "typedoc": "0.15.2"
+    "typedoc": "0.15.2",
+    "typescript": "~4.1.2"
```

Comment:
@witemple-msft this is a duplication (similar to what we have in the eslint plugin) but it should be fine.

Reply:
It feels odd to me too, but I don't think it'll hurt anything?